src/main/kotlin/bandcampcollectiondownloader/Args.kt

```diff
@@ -41,6 +41,15 @@ data class Args(
         var jobs: Int = 4,
 
         @CommandLine.Option(names = ["-v", "--version"], versionHelp = true, description = ["Display the version and exits."])
-        var version: Boolean = false
+        var version: Boolean = false,
+        
+        @CommandLine.Option(names = ["-ft", "--filter-title"], usageHelp = false, description = ["only downloads releases matching the specified string"])
+        var filterTitle: String? = null,
 
-)
\ No newline at end of file
+        @CommandLine.Option(names = ["-fa", "--filter-artist"], usageHelp = false, description = ["only downloads releases by the specified artist"])
+        var filterArtist: String? = null,
+
+        @CommandLine.Option(names = ["-fd", "--filter-date"], usageHelp = false, description = ["only downloads releases bought after the specified ISO8601"])
+        var filterDate: String? = null
```

Comment:
Here I believe we can be a little more precise than `String` and directly ask PicoCLI to check for a valid `Date`. This way any PicoCLI would stop the execution very early if the provided argument was not a date, and thus you would not have to worry later on whether the argument was a well formatted date or not. See this part of the PicoCLI documentation: https://picocli.info/#_strongly_typed_everything. WDYT?